client/index.html

```diff
@@ -1,9 +1,17 @@
 <html>
   <head>
+    <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=1.0,user-scalable=no">
     <script src="/index.js"></script>
     <link rel="stylesheet" type="text/css" href="/index.css">
   </head>
+  <div id="subject-wrapper">
```

Comment:
These wrappers should be included in `main`

Reply:
Fixed, wrappers are renamed to `container`s.